kentikapi/internal/api_payloads/saved_filter.go

```diff
@@ -0,0 +1,182 @@
+package api_payloads
+
+import (
+	"time"
+
+	"github.com/kentik/community_sdk_golang/kentikapi/internal/utils"
+	"github.com/kentik/community_sdk_golang/kentikapi/models"
+)
+
+type GetAllSavedFilterResponse []GetSavedFilterResponse
+
+func (p GetAllSavedFilterResponse) ToSavedFilters() ([]models.SavedFilter, error) {
+	var result []models.SavedFilter
+	err := utils.ConvertList(p, GetSavedFilterResponse.ToSavedFilter, &result)
+	if err != nil {
+		return nil, err
+	}
+	return result, err
+}
+
+type GetSavedFilterResponse struct {
+	SavedFilterPayload
+}
+
+type CreateSavedFilterRequest struct {
+	SavedFilterPayload
+}
+
+type CreateSavedFilterResponse struct {
+	SavedFilterPayload
+}
+
+type UpdateSavedFilterRequest struct {
+	SavedFilterPayload
+}
+
+type UpdateSavedFilterResponse struct {
+	SavedFilterPayload
+}
+
+type SavedFilterPayload struct {
```

Comment:
Nit: I think that payload structs, such as SavedFilterPayload could (and should) be unexported in order to reduce package API surface.

Reply:
Done